app/views/tracks/show/joined.html.haml

```diff
@@ -0,0 +1,66 @@
+#page-track-dashboard
+  %nav.track-nav
+    .lg-container.container
+      = link_to "<-", tracks_path, class: "back", 'aria-label': "Back to all tracks"
+      = image_tag(@track.icon_url, class: 'c-track-icon')
+      .title= @track.title
+      .tabs
+        = link_to "#", class: 'selected' do
+          Overview
+
+        = link_to track_concepts_path(@track) do
+          Concepts
+
+        = link_to "#" do
+          Exercises
+
+  %header.track-header
+    .lg-container.container
+      = image_tag(@track.icon_url, class: 'c-track-icon')
+      .info
+        %h1= @track.title
+        .joined
+          Joined on
+          %time{ datetime: @user_track.created_at.iso8601 }
+            = format_date(@user_track.created_at)
+      .progress-bar
+
+  %article
+    .lg-container.container
+      .lhs
+      .rhs
+        %section.contributors
+          %header
+            %h2 Contributors
+            .count 123
+          .showcase
+            = link_to "#" do
+              .img{ style: "background-image:url(https://avatars3.githubusercontent.com/u/135246?s=460&u=733b8575c8f0dbb5a52840c3efb2480b166c897a&v=4)" }
+              .name Erik Schierboom
+              .reputation
+                = reputation_icon(color: "#8480A0")
```

Comment:
This has been updated to #6D6986 in the designs (`text-textLight`in Tailwind, or `Light / Label Secondary` in Figma).